### PR TITLE
Add version flag, update dependencies

### PR DESCRIPTION
**Pull request by dmitryk-dk**
Enable version flag 
Screenshot 2023-09-28 at 11 19 58

Updated dependencies for the backend plugin

Related issue: https://github.com/VictoriaMetrics/grafana-datasource/issues/68

Signed-off-by: dmitryk-dk [d.kozlov@victoriametrics.com](mailto:d.kozlov@victoriametrics.com)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,46 +1,63 @@
 module github.com/VictoriaMetrics/grafana-datasource
 
-go 1.19
+go 1.20
```
Comment (denisgolius) [inline go.mod:3]: What about go 1.21 ?

**Review by denisgolius — COMMENTED** [review]
Only the one question stop me from approving =)

File: go.mod (hunk 1 above)
Comment (dmitryk-dk) [reply go.mod]: updated go version

**Review by denisgolius — APPROVED** [review]
LGTM!

File: pkg/buildinfo/version.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package buildinfo
+
+import (
+	"flag"
+	"fmt"
+	"os"
+)
+
+var version = flag.Bool("version", false, "Show VictoriaMetrics version")
```
Comment (hagen1778) [inline pkg/buildinfo/version.go:9]: This should be updated accordingly

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 ## tip
 
+* FEATURE: add a `--version` flag for backend datasource, update backend plugin to a latest release version. See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/68).
```
Comment (hagen1778) [inline CHANGELOG.md:5]:
```suggestion
* FEATURE: add `--version` flag for backend datasource binary. See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/68).
```

**Review by hagen1778 — CHANGES_REQUESTED** [review]

File: pkg/buildinfo/version.go (hunk 2 above)
Comment (dmitryk-dk) [reply pkg/buildinfo/version.go]: updated

File: CHANGELOG.md (hunk 3 above)
Comment (dmitryk-dk) [reply CHANGELOG.md]: updated

**Review by hagen1778 — APPROVED** [review]

**Pull request merged**